### ISSUE TITLE
CookieManager affects other st components behaviour.

### ISSUE THREAD

**Issue by aegeavaz** [Mohamed-512/Extra-Streamlit-Components#25]
Hi...
Someone post it also here: https://discuss.streamlit.io/t/cookies-support-in-streamlit/16144/35
but the solution provided in this post is forcing to use forms and that shouldn't be the solution.

```
import streamlit as st
import extra_streamlit_components as stx

cookie_manager = stx.CookieManager()

button = st.button("Get cookies")
if button:
    st.subheader("All Cookies:")
    cookies = cookie_manager.get_all()
    st.write(cookies)
    st.success("This should show up for longer than a split second")
```

The problem is that the st.success message dissapears....
Any solution for this issue?

Thank you so much.
Regards.

**Comment by benlindsay** [comment]
@aegeavaz I think this is related to my issue I just created: https://github.com/Mohamed-512/Extra-Streamlit-Components/issues/58

**Comment by Mohamed-512** [comment]
Should be fixed in 0.1.70
